src/Service/ConfigurationService.php

```diff
@@ -345,4 +346,25 @@ public function getAdyenGivingCharityWebsite(string $salesChannelId)
             $salesChannelId
         );
     }
+
+    /**
+     * @param string $salesChannelId
+     * @return array|mixed|null
+     */
+    public function getDomainUrlId(string $salesChannelId): ?string
+    {
+        return $this->systemConfigService->get(self::BUNDLE_NAME . '.config.returnUrlId', $salesChannelId);
+    }
+
+    /**
+     * @param string $salesChannelId
+     * @return null|bool
+     */
+    public function getEnableOverrideDefaultDomain(string $salesChannelId): ?bool
```

Comment:
Since it is checking the `bool` state of the config field `enableOverrideDefaultDomain`, changing the name of the method as `getIsOverrideDefaultDomainEnabled` may be consistent.